DDG4/python/DDSim/Helper/Action.py

```diff
@@ -30,6 +30,22 @@ class Action(ConfigHelper):
   >>> SIM = DD4hepSimulation()
   >>> SIM.action.mapActions['ecal'] =( "CaloPreShowerSDAction", {"FirstLayerNumber": 1} )
 
+  Additional actions can be set as well with the following syntax variations:
+
+  >>> SIM = DD4hepSimulation()
+  # single action by name only:
+  >>> SIM.action.run = "Geant4TestRunAction"
+  # multiple actions with comma-separated names:
+  >>> SIM.action.event = "Geant4TestEventAction/Action0,Geant4TestEventAction/Action1"
```

Comment:
I think we should just add the `/Action` ourselves when we create the action, that unifies the setting of a single or multiple actions.

Reply:
We should probably also allow a list of strings `["MyFirstGeant4TestEventAction", "MySecondGeant4TestEventAction"]`

Reply:
The issue with a list of strings is that it interferes with the list of dicts if we just use `isinstance` checks... It would make `makeListOfDictFromJSON` more complicated: instead of
```python
    if isinstance(val, list):
      # assumes: [ { "name": "Geant4TestEventAction", "parameter": {"Property_int": 10} } ]
      return val
```
it would become
```
    if isinstance(val, list):
      if isinstance(val[0], str):
        # assumes: [ "Geant4TestEventAction", "Geant4TestEventAction" ]
        return [dict(name=v) for v in val]
      if isinstance(val[0], dict):
        # assumes: [ { "name": "Geant4TestEventAction", "parameter": {"Property_int": 10} } ]
        return val
```
and then we need to check that `val != []`... I guess not that bad.

Reply:
Resolved in 62b7b14e6f2c764e17ee862be8dfc24b8f6cb773.